types/shared.d.ts

```diff
@@ -6,6 +6,7 @@ export interface PageData {
   relativePath: string
   title: string
   titleTemplate?: string | boolean
+  nav: boolean
```

Comment:
I don't think it should be replaced into PageData, theme could use it from frontmatter directly

Reply:
Good idea!